cumulus_library/studies/core/builder_core_medication.py

```diff
@@ -113,12 +99,17 @@ def prepare_queries(self, cursor: object, schema: str) -> dict:
         :param schema: the schema/db name, matching the cursor
 
         """
-        medication_datasources = self._check_data_in_fields(cursor, schema)
+        medication_datasources, missing_userselected = self._check_data_in_fields(
```

Comment:
nit: one more of these and maybe just return a dict and pass it down like `**query_args`